src/detect-content.c

```diff
@@ -402,6 +400,16 @@ bool DetectContentPMATCHValidateCallback(const Signature *s)
 
     uint32_t max_right_edge = (uint32_t)max_right_edge_i;
 
+    int min_dsize_required = SigParseMaxRequiredDsize(s);
```

Comment:
interesting mismatch in variable and function name

Reply:
Hehe ... the max required value is the minimal acceptable value :-)